### ISSUE TITLE
Possibility of CV ports?

### ISSUE THREAD

**Issue by mxmilkiib** [lvtk/jlv2#4]
DPF [allows](https://github.com/DISTRHO/DPF/search?utf8=%E2%9C%93&q=cv&type=) a [CV](https://linuxmusicians.com/viewtopic.php?f=1&t=20701) property hint to turn input/output ports into CV ports.

Might it be possible for jlv2 to allow something similar?

**Comment by dromer** [comment]
Does JUCE even have the notion of CV?

I think everyone just abuses plain audio ports for CV-like functionality (so without offset or range metadata). This may need `cv <-> audio` conversion inside jlv2 then.

**Comment by mfisher31** [comment]
Nope, no sense of CV.  Need a good way to handle it, but don't know how yet.  Might have to make the AudioProcessor subclass public, with custom "process" function.  THen calling code can use that, or just not support CV.

**Comment by falkTX** [comment]
I have made quite a lot of changes to my local "juce" copy so the graph could support CV.
The internal patchbay of carla uses an old copy of juce when it was still GPLv2, heavily modified to remove all gui stuff and anything not needed to make it work as the super-base-support for carla.

Feel free to take any ideas, relevant code is at https://github.com/falkTX/Carla/tree/master/source/modules/water/processors
But I have to say it is not straight forward at all.. JUCE makes a lot of assumptions for audio.
(also for a single midi port per node, which is annoying but havent resolved that yet)

**Comment by mfisher31** [comment]
Thanks for the tip @falkTX !
